client/post-editor/editor-gutenberg-blocks-warning-dialog/index.jsx

```diff
@@ -78,9 +94,29 @@ class EditorGutenbergBlocksWarningDialog extends Component {
 
 	useClassicEditor = () => {
 		this.props.useClassic();
-		this.setState( {
-			forceClassic: true,
-		} );
+		this.redirectToWpAdmin();
+	};
+
+	redirectToWpAdmin = () => {
+		const { isAtomic, isPrivate, postId, postType, buildSiteAdminUrl } = this.props;
+		if ( isAtomic && isPrivate ) {
```

Comment:
I've only had a quick look at this, but if this is `false` should we be calling `setState( { forceClassic: true } )`?

I'm thinking of whether we need to retain the existing `useClassicEditor()` behaviour. Although we wouldn't want to call `setState( { forceClassic: true } )` if `redirectToWpAdmin()` was being called from `cDM` 🤔

Reply:
Good note! I tweaked it slightly